pallets/msa/src/types.rs

```diff
@@ -296,3 +296,11 @@ impl<T: Config> PermittedDelegationSchemas<T>
 		Ok(())
 	}
 }
+
+/// Helper function to compute CID of given bytes and return Vec<u8>
+pub fn compute_cid(bytes: &[u8]) -> Vec<u8> {
+	let cid = compute_cid_v1(bytes).expect("Failed to compute CID");
```

Comment:
This can panic if suddenly used in the pallet code. Please either gate it with limited feature flags for `benchmarks` and `tests` or just return a Result